tests/compiler/std/array.ts

```diff
@@ -585,6 +585,12 @@ arr.push(3);
 var filteredArr: i32[] = arr.filter((value: i32, index: i32, array: Array<i32>): bool => value >= 2);
 assert(filteredArr.length == 2);
 
+// Array#find ////////////////////////////////////////////////////////////////////////////////////
+var found: i32 | null = arr.find((value: i32, index: i32, array: Array<i32>): bool => value >= 2);
```

Comment:
@LiaoPeng or you can use `findIndex()` and look for `<i32>-1` I've made an effort to implement this function already on another pull request.

Reply:
@jtenner It can't distinguish the difference between `null` and  `0`,  I am not sure the suggestion work.

Reply:
There is no difference between `null` and `0` in AS (when compiling for WASM32), quite similar to C/C++. Might make sense to implement a compile-time check to avoid mixing them, though.